wcomponents-test-lib/src/test/java/com/github/bordertech/wcomponents/test/WTabSetWebElementTest.java

```diff
@@ -0,0 +1,42 @@
+package com.github.bordertech.wcomponents.test;
+
+import com.github.bordertech.wcomponents.test.components.WTabSetUI;
+import com.github.bordertech.wcomponents.test.selenium.ByTabSet;
+import com.github.bordertech.wcomponents.test.selenium.MultiBrowserRunner;
+import com.github.bordertech.wcomponents.test.selenium.SeleniumJettyTestCase;
+import com.github.bordertech.wcomponents.test.selenium.driver.SeleniumWComponentsWebDriver;
+import com.github.bordertech.wcomponents.test.selenium.element.SeleniumWComponentWebElement;
+import com.github.bordertech.wcomponents.test.selenium.element.SeleniumWTabSetWebElement;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(MultiBrowserRunner.class)
+public class WTabSetWebElementTest extends SeleniumJettyTestCase {
+
+	/**
+	 * Constructor...
+	 *
+	 */
+	public WTabSetWebElementTest() {
+		super(new WTabSetUI());
+	}
+
+	@Test
+	public void testTabFeatures() {
+		SeleniumWComponentsWebDriver driver = getDriver();
+
+		SeleniumWTabSetWebElement wTabSet = driver.findWTabSet(new ByTabSet());
+		assertEquals(wTabSet.findSelectedTabLabel(), "First Tab");
+		SeleniumWComponentWebElement selectedTabContent = wTabSet.findSelectedTabContent();
+		assertEquals(selectedTabContent.getText(), "Tab: One");
+		wTabSet.clickTab("Second Tab");
+
+		assertEquals(wTabSet.findSelectedTabLabel(), "Second Tab");
+		selectedTabContent = wTabSet.findSelectedTabContent();
+		assertEquals(selectedTabContent.getText(), "Tab: Two");
+
+
+	}
+}
```

Comment:
How would you test the exceptions you wrote into WTabSetWebElement and can you run the tests with multiple WTabSets with each of the WTabSetType values - refer to my comments about WTabSetType.ACCORDION.